### DISCUSSION TITLE
HC_envSet/Crypto/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#89] [Show and tell]
# HC_envSet/Crypto/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://hello-ctf.com/HC_envSet/Crypto/

**Comment by Strawberry47** [comment]
Web环境设置、MISC环境设置打不开啦

**Reply by Randark-JMT** [reply]
Web 和 Misc 的环境设置部分还没写，建议找作者 @ProbiusOfficial 进行催更

**Comment by Xia0-0** [comment]
之前，在环境配置与工具安装上花费大量时间，然后到现在没用到，哭死，所以建议配置用到在配^O^，😂😂
